frontend/packages/console-shared/src/selectors/node.ts

```diff
@@ -48,3 +48,5 @@ export const getNodeSecondaryStatus = (node: NodeKind): string[] => {
   }
   return states;
 };
+
+export const getNodeTaints = (node: NodeKind) => _.get(node, 'spec.taints', []);
```

Comment:
Avoid `_.get` because it's not type safe. Remove this and inline `node.spec.taints` where you use it.